test/integration/visitor_can_view_deal_page_test.rb

```diff
@@ -2,17 +2,17 @@
 
 class VisitorCanViewDealPageTest < ActionDispatch::IntegrationTest
   include CategoryItemsSetup
-  test "visitor can view the daily deal page" do
+  test 'visitor can view the daily deal page' do
     create_categories_and_items
     create_featured_item
     visit '/deal'
 
-    assert page.has_content?("Today's Gnarliest Deal!")
-    assert page.has_content?("gnar possum")
-    assert page.has_content?("$1000")
+    assert page.has_content?("Today's Gnarliest Item!")
+    assert page.has_content?('gnar possum')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.